Gopkg.toml

```diff
@@ -116,3 +116,15 @@
 [[constraint]]
   name = "github.com/hashicorp/go-plugin"
   branch = "master"
```

Comment:
let's pin this to the specific revision we're using now - so instead of `branch = "master"`, use `revision = "3f118e8ee104b6f22aeb12453fab56aed1356186"`